### PR TITLE
Belated metadata test.

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala [hunk 1]
```diff
@@ -184,5 +185,51 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
...
+      metadata.outputs shouldBe defined
+      metadata.outputs.get should have size 5
+      metadata.calls should have size 5
+
+      // ok if this explodes, it's a test
+      val devOutputs = metadata.outputs.get.get("w.A.A_out").get
+      val wdlArray = devOutputs.asInstanceOf[WdlArray]
+      wdlArray.wdlType shouldBe WdlArrayType(WdlStringType)
+
+      (wdlArray.value map { case WdlString(string) => string }) shouldEqual Vector("jeff", "chris", "miguel", "thibault", "khalid", "scott")
```
Comment (geoffjentry) [inline src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala]: Poor ruchi doesn't get a mention :(

**Comment by geoffjentry** [comment]
I love tests!

File: src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala [hunk 2]
```diff
@@ -184,5 +185,51 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
...
+        call.returnCode.get shouldBe 0
+        call.stdout shouldBe defined
+        call.stderr shouldBe defined
+        call.inputs should have size 1
+        call.backend.get shouldEqual "Local"
+        call.backendStatus should not be defined
+        call.executionStatus shouldBe "Done"
+      }
+
+      (devCalls map { _.outputs.get.get("C_out").get.asInstanceOf[WdlInteger].value } ) shouldEqual Vector(400, 500, 600, 800, 600, 500)
```
Comment (kshakir) [inline src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala]: Should this be `})` instead of `} )`?

**Comment by kshakir** [comment]
Nominating @geoffjentry as second reviewer, who already reviewed I believe. So bouncing back to @mcovarr.

**Comment by geoffjentry** [comment]
@kshakir I try to review everything although sometimes I just silently approve. whether or not that ever counts as one of the official two is up to you all :)

File: src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala (hunk 2 above)
Comment (mcovarr) [inline src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala]: When I hit this with an IntelliJ formatting I get `})`, so I'll change to that.  I also get an expression smeared out over 3 lines, which I will not change to.  :smile:

File: src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala (hunk 1 above)
Comment (mcovarr) [inline src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala]: If I wasn't using a preexisting WDL I'd fix that.  :smile:

**Pull request merged**